docs/testing/simulation-tests.md

```diff
@@ -4,12 +4,134 @@ sidebar_position: 2
 
 # Simulation Tests
 
-<!-- Difference between unit and simulation tests -->
+**Note**: simulation tests are being deprecated in favor of [the Sandbox](sandbox.md). Until the Sandbox is ready, simulation tests are still a useful approach to the types of testing described on this page. Please note, however, that there will not be ongoing development for it.  
```

Comment:
the sandbox link whiffs, want to just add an empty template for now for this page so that they don't get the 404 page?

Reply:
Ah thanks, I thought I removed all these but missed this one. Will fix